### PR TITLE
 #72: Adds apollo-client local state management and migrates to hashrouter

### PR DESCRIPTION
**Pull request by utkarsh867**

### CODE REVIEW

File: src/components/main/BookingSummary.js [hunk 1]
```diff
@@ -66,16 +72,21 @@ const BookingSummary = ({
   });
 
   const handleOnConfirmPress = async () => {
-    date = moment(date).startOf('day');
-    start = moment(date).add(Number(start.substring(0, 2)), 'hour');
-    end = moment(date).add(Number(end.substring(0, 2)), 'hour');
```
Comment (utkarsh867) [inline src/components/main/BookingSummary.js:71]: @98sean98 This is very unacceptable.

Comment (98sean98) [reply src/components/main/BookingSummary.js:71]: I'm fixing

**Comment by 98sean98** [comment]
closes #72

**Comment by 98sean98** [comment]
will need to endurance test this first before merging

**Review by welvin21 — APPROVED** [review]

**Comment by utkarsh867** [comment]
@98sean98 Can you please test the changes in the application?

**Pull request merged**